src/core/thememgr.cpp

```diff
@@ -37,10 +38,10 @@ QString ThemeMgr::getIconFile(const QString &p_icon) const
     }
 
     // If there is an ICONS folder in the theme configuration, use the custom ICONS from it.
-    QString customIcons = this->customIconsPath + p_icon;
-    QFile iconsFile(customIcons);
-    if (iconsFile.exists()) {
-        return customIcons;
+    QString customIcon = getFile(Theme::File::Icon) + "/" + p_icon;
+    QFile customIconFile(customIcon);
```

Comment:
Use `QFile::exists()` static version, which is faster and light weight.

Reply:
done